app/data/models/delinkedCalculation.js

```diff
@@ -0,0 +1,41 @@
+module.exports = (sequelize, DataTypes) => {
+  const number2 = 2
+  const number15 = 15
+  const delinkedCalculation = sequelize.define('delinkedCalculation', {
+    applicationId: { type: DataTypes.INTEGER, allowNull: false },
+    calculationId: { type: DataTypes.INTEGER, primaryKey: true, allowNull: false },
+    sbi: { type: DataTypes.INTEGER, allowNull: false },
+    frn: { type: DataTypes.BIGINT, allowNull: false },
```

Comment:
is there a specific reason to use bigInt ?

Reply:
bigInt is the format we use for all the other services on FRN so I kept the same

Reply:
although I agree that we are using it as bigInt elsewhere, this is not the best implementation because :

For the given numbers frn: 1234567890 and sbi: 123456789, it is more efficient to use the int data type because:

Both values fit comfortably within the range of a 32-bit integer.
int uses less storage space and provides better performance compared to bigint.

this is a non-mutable field(I hope we dont change it dynamically), so it's never going to reach upper boundary of 32-bit

Reply:
changed to integer